.github/workflows/main.yml

```diff
@@ -177,10 +177,26 @@ jobs:
           npm run build
         shell: bash
 
+      - name: Decide on ref of CodeQL repo
+        id: choose-ref
+        shell: bash
+        run: |
+          if [[ "${{ matrix.version }}" == "nightly" ]]
+          then
+            REF="main"
+          elif [[ "${{ matrix.version }}" == "v2.2.6" || "${{ matrix.version }}" == "v2.3.3" ]]
+          then
```

Comment:
It's confusing that different versions are being used. Off the top of my head, I don't know if this is correct. But if you've checked the version numbers then this looks fine.

Reply:
These are indeed the two versions from [the ones we test against](https://github.com/github/vscode-codeql/blob/3536124fbc597fe282d539d1863e7baf27e443bb/.github/workflows/main.yml#L154) that are too old to have a tag on CodeQL. We could try to do something more clever for these like finding a ref from around the time they were released, but since it works like this and we won't be testing against them forever, I think this is fine.